### ISSUE TITLE
Simple dynamic programming problem

### ISSUE THREAD

**Issue by azev77** [hnlee77/ApproximateDPs.jl#32]
Hi, is it possible to use your package to solve the simple dynamic programming problem here:
https://discourse.julialang.org/t/solving-the-4-quadrants-of-dynamic-optimization-problems-in-julia-help-wanted/73285

**Comment by JinraeKim** [comment]
@azev77 I don't think that it is a good idea to use this package for solving decent problems.
This package was in progress and has not been maintained for a while (I'm pretty sure that this package would not probably be working now).
@hnlee77 Could you answer his question more?
